core/modules/wsched/ChunkDisk.h

```diff
@@ -40,30 +40,32 @@
 #include "memman/MemMan.h"
 #include "proto/worker.pb.h"
 #include "wbase/Task.h"
-
+#include "wsched/ChunkTaskCollection.h"
 
 namespace lsst {
 namespace qserv {
 namespace wsched {
 
 /// Limits Tasks to running when resources are available.
 /// TODO: DM-4943 Maybe merge this class into ScanScheduler.
-class ChunkDisk {
+class ChunkDisk : public ChunkTaskCollection {
```

Comment:
I expect to delete this class, but I wanted to keep it around for testing purposes to make sure that ChunkTaskList is better. ChunkTaskCollection should probably go away at the same time as it is only intended to make it easy to switch between ChunkDisk and ChunkClassList. I'd rather avoid writing documentation for code that I believe is doomed.